Create Pega Profile Export destination

paratAtPega opened the pull request:
Initial merge of Pega Profile Export for destination

paratAtPega committed:
Create Pega.md  file Profile Export feature

paratAtPega committed:
Rename pega,md to pega.md

paratAtPega committed:
Updated

paratAtPega committed:
Updated

paratAtPega committed:
Updated

paratAtPega committed:
Updated

paratAtPega committed:
updated

paratAtPega committed:
Updated

paratAtPega committed:
update

paratAtPega committed:
Update pega.md

paratAtPega committed:
Update pega.md

paratAtPega committed:
Updated

paratAtPega committed:
Update pega.md

paratAtPega committed:
Update pega.md

paratAtPega committed:
Update pega.md

paratAtPega committed:
Update pega.md

paratAtPega committed:
Create pega-profile.md

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Delete pega.md

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Create test.txt

paratAtPega committed:
profile data image

paratAtPega committed:
Delete test.txt

paratAtPega committed:
Add files via upload

paratAtPega committed:
Delete pega-profile-data.png.png

paratAtPega committed:
Add files via upload

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Add files via upload

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Delete pega-profile-connect-destination.png.png

paratAtPega committed:
Add files via upload

paratAtPega committed:
Merge branch 'AdobeDocs:main' into create-destination

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Update pega-profile.md

vjeffs2 committed:
Update pega-profile.md

paratAtPega committed:
Merge pull request #1 from vjeffs2/patch-2

Update pega-profile.md

paratAtPega committed:
Updated with template defaults and adobe defaults

paratAtPega committed:
Updates from doc team

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Update pega-profile.md

vgiurgiu commented:
Hi @paratAtPega - I see that you've been making several updates and edits. Please let us know when the documentation is ready to review.

paratAtPega commented:
Yes, Please review.

Thanks

Tony

From: Victor Giurgiu ***@***.***>
Sent: Thursday, November 3, 2022 4:51 PM
To: AdobeDocs/experience-platform.en ***@***.***>
Cc: Parakka, Tony ***@***.***>; Mention ***@***.***>
Subject: ExtMsg: Re: [AdobeDocs/experience-platform.en] Create Pega Profile Export destination (PR #131)


CAUTION: This email was sent from an EXTERNAL source. Think before clicking links or opening attachments.

Hi @paratAtPega<https://github.com/paratAtPega> - I see that you've been making several updates and edits. Please let us know when the documentation is ready to review.

—
Reply to this email directly, view it on GitHub<https://github.com/AdobeDocs/experience-platform.en/pull/131#issuecomment-1302648689>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AU3DX2BRS357KKAB4V63KUTWGQQTJANCNFSM6AAAAAARPSZTFI>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
```
missing pipe character at beginning of line. this will lead to a broken table.

paratAtPega replied:
Added

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+---
+title: Pega Profile Connector
+description: Use the Pega Profile Connector for S3 in Adobe Experience Platform to export full or incremental, or both, profile data to Amazon S3 cloud storage. In Pega Customer Decision Hub, data jobs can be scheduled in Customer Profile Designer to import profile data periodically from Amazon S3 storage. 
```
minor, I would add Amazon S3 at first mention in the description.

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+
+## Overview {#overview}
+
+Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer  ...
```
some minor comments: 

we can break the paragraph in two parts for easier consumption: 

Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer Decision Hub profile.

This connector helps set up the initial export of profile data and also helps sync up new profiles periodically into Pega Customer Decision Hub.  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.

also, there are some missing DNL tags. it would be great if you could add them, for consistency.

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+
+### Use case 1
+
+A marketer wants to initially setup Pega Customer Decision Hub with profile data loaded from Adobe Experience Platform. This is an initial full load followed by delta loads on a scheduled basis.
```
minor, but I believe setup should be spelled as "set up" here, since it is not a noun.

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+
+## Prerequisites {#prerequisites}
+
+Before you can use this destination to export data out of Adobe Experience Platform and import profiles into Pega [!DNL Pega Customer Decision Hub], make sure you complete the following prerequisites:
```
minor: ~Pega~ [!DNL Pega Customer Decision Hub]

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+
+For [!DNL Pega Profile Connector] destinations, [!DNL Platform] creates a `.csv` file in Amazon S3 storage location that you provided. For more information about the files, see [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) in the segment activation tutorial.
+
+A successful import of profile data from S3 inserts data in the Pega Customer profile datastore. The imported Customer profile data can be validated in Pega Customer Profile Designer , as shown in the following figure.
```
minor: lowercase C in Customer profile data

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+A successful import of profile data from S3 inserts data in the Pega Customer profile datastore. The imported Customer profile data can be validated in Pega Customer Profile Designer , as shown in the following figure.
+![Image of the UI screen where you can validate Adobe profile data in Customer Profile Designer](../../assets/catalog/personalization/pega-profile/pega-profile-data.png)
+
+In Pega Customer Decision Hub, Data Administrators can configure data jobs in Customer Profile Designer to import profile data periodically from S3 as shown in the following figure.
```
is there maybe a Pega documentation page that we could link out to here, which provides more details about what users need to do in Pega to configure data jobs to import profile data from Amazon S3?

Note, I just saw the link (https://academy.pega.com/topic/import-data-jobs/v1) further below. Maybe we can add a sentence here, like: See the `[additional resources](#additional-resources)` for more information about how to configure data jobs to import profile data from Amazon S3

vgiurgiu replied:
also, I believe Data Administrators can be lowercase. unless it is a branded term in Pega?

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,134 @@
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+See [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.
+
```
there is a subsection which would come in handy here, about the mapping step. see the self-service documentation template: 

https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#map

Could we perhaps add this section and briefly describe the particularities of the mapping step for this new Pega destination?

paratAtPega replied:
Mapping need to be done in Pega Customer Decision Hub side. I tried adding few predefined fields in AEP side, however, the connector complains that the source array field cannot be mapped to String. Also, if I use predefined Attributes, I was not able to create custom target field. so revert back my changes

vgiurgiu replied:
@paratAtPega - Apologies, I missed your reply (including this comment). Perhaps you can reach out to our partnership team to inquire about the error you are seeing? 

Also, just wondering, since this is a file export integration, I would think that users will export files that include attributes and maybe also identities. Is that correct? I think it would be valuable to show that in a mapping step in the documentation, similar to what other destinations have done (see examples in https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#map).

Also, apart from this comment, the documentation looks great. We'll publish this documentation page after the Pega team submits the publish request for the destination and our team approves it.

paratAtPega replied:
Added mapping paragraph after referencing other file based connectors. (google-cloud)

vgiurgiu reviewed (commented):
Thanks for opening the PR! Excellent work! I left just a few comments, mostly minor. 
Let me know please if you have any questions.

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Updated review comments

paratAtPega commented:
Hi Victor,
         Thanks for the review comments and I have added the changes.


Thanks

Tony

From: Victor Giurgiu ***@***.***>
Sent: Friday, November 4, 2022 5:15 PM
To: AdobeDocs/experience-platform.en ***@***.***>
Cc: Parakka, Tony ***@***.***>; Mention ***@***.***>
Subject: ExtMsg: Re: [AdobeDocs/experience-platform.en] Create Pega Profile Export destination (PR #131)


CAUTION: This email was sent from an EXTERNAL source. Think before clicking links or opening attachments.

@vgiurgiu commented on this pull request.

Thanks for opening the PR! Excellent work! I left just a few comments, mostly minor.
Let me know please if you have any questions.

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014463645>:

> +## Supported identities {#supported-identities}

+

+[!DNL Pega Customer Decision Hub] supports the activation of custom user IDs described in the table below. For more details, see [identities](/help/identity-service/namespaces.md).

+

+|Target Identity|Description|

+|---|---|

+|*CustomerID*|Common User Identifier that uniquely identifies a profile in [!DNL Pega Customer Decision Hub] and Adobe Experience Platform|

+

+{style="table-layout:auto"}

+

+## Export type and frequency {#export-type-frequency}

+

+Refer to the table below for information about the destination export type and frequency.

+

+| Item | Type | Notes |

+---------|----------|---------|

missing pipe character at beginning of line. this will lead to a broken table.

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014463904>:

> @@ -0,0 +1,134 @@

+---

+title: Pega Profile Connector

+description: Use the Pega Profile Connector for S3 in Adobe Experience Platform to export full or incremental, or both, profile data to Amazon S3 cloud storage. In Pega Customer Decision Hub, data jobs can be scheduled in Customer Profile Designer to import profile data periodically from Amazon S3 storage.

minor, I would add Amazon S3 at first mention in the description.

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014466373>:

> @@ -0,0 +1,134 @@

+---

+title: Pega Profile Connector

+description: Use the Pega Profile Connector for S3 in Adobe Experience Platform to export full or incremental, or both, profile data to Amazon S3 cloud storage. In Pega Customer Decision Hub, data jobs can be scheduled in Customer Profile Designer to import profile data periodically from Amazon S3 storage.

+---

+

+# Pega Profile Connector

+

+## Overview {#overview}

+

+Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer Decision Hub profile. This connector helps setup initial export of profile data and also helps sync up new profiles periodically into Pega Customer Decision Hub.  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.

some minor comments:

we can break the paragraph in two parts for easier consumption:

Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer Decision Hub profile.

This connector helps set up the initial export of profile data and also helps sync up new profiles periodically into Pega Customer Decision Hub. Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.

also, there are some missing DNL tags. it would be great if you could add them, for consistency.

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014467113>:

> +

+## Overview {#overview}

+

+Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the Pega Customer Decision Hub profile. This connector helps setup initial export of profile data and also helps sync up new profiles periodically into Pega Customer Decision Hub.  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.

+

+>[!IMPORTANT]

+>

+>This documentation page was created by Pegasystems. For any inquiries or update requests, please contact Pega directly ***@***.***).

+

+## Use cases

+

+To help you better understand how and when you should use the [!DNL  Pega Profile Connector] destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.

+

+### Use case 1

+

+A marketer wants to initially setup Pega Customer Decision Hub with profile data loaded from Adobe Experience Platform. This is an initial full load followed by delta loads on a scheduled basis.

minor, but I believe setup should be spelled as "set up" here, since it is not a noun.

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014467535>:

> +

+## Use cases

+

+To help you better understand how and when you should use the [!DNL  Pega Profile Connector] destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.

+

+### Use case 1

+

+A marketer wants to initially setup Pega Customer Decision Hub with profile data loaded from Adobe Experience Platform. This is an initial full load followed by delta loads on a scheduled basis.

+

+### Use case 2

+

+A marketer wants up-to-date profile data from Adobe Experience Platform available in Pega Customer Decision Hub that enhances the Pega insights around customer profiles on an ongoing basis.

+

+## Prerequisites {#prerequisites}

+

+Before you can use this destination to export data out of Adobe Experience Platform and import profiles into Pega [!DNL Pega Customer Decision Hub], make sure you complete the following prerequisites:

minor: Pega [!DNL Pega Customer Decision Hub]

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014468824>:

> +

+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.

+

+## Activate segments to this destination {#activate}

+

+>[!IMPORTANT]

+>

+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.

+

+See [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.

+

+## Exported data / Validate data export {#exported-data}

+

+For [!DNL Pega Profile Connector] destinations, [!DNL Platform] creates a `.csv` file in Amazon S3 storage location that you provided. For more information about the files, see [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) in the segment activation tutorial.

+

+A successful import of profile data from S3 inserts data in the Pega Customer profile datastore. The imported Customer profile data can be validated in Pega Customer Profile Designer , as shown in the following figure.

minor: lowercase C in Customer profile data

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014469400>:

> +## Activate segments to this destination {#activate}

+

+>[!IMPORTANT]

+>

+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.

+

+See [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.

+

+## Exported data / Validate data export {#exported-data}

+

+For [!DNL Pega Profile Connector] destinations, [!DNL Platform] creates a `.csv` file in Amazon S3 storage location that you provided. For more information about the files, see [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) in the segment activation tutorial.

+

+A successful import of profile data from S3 inserts data in the Pega Customer profile datastore. The imported Customer profile data can be validated in Pega Customer Profile Designer , as shown in the following figure.

+![Image of the UI screen where you can validate Adobe profile data in Customer Profile Designer](../../assets/catalog/personalization/pega-profile/pega-profile-data.png)

+

+In Pega Customer Decision Hub, Data Administrators can configure data jobs in Customer Profile Designer to import profile data periodically from S3 as shown in the following figure.

is there maybe a Pega documentation page that we could link out to here, which provides more details about what users need to do in Pega to configure data jobs to import profile data from Amazon S3?

Note, I just saw the link (https://academy.pega.com/topic/import-data-jobs/v1<https://academy.pega.com/topic/import-data-jobs/v1>) further below. Maybe we can add a sentence here, like: See the [additional resources](#additional-resources) for more information about how to configure data jobs to import profile data from Amazon S3

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014469666>:

> +## Activate segments to this destination {#activate}

+

+>[!IMPORTANT]

+>

+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.

+

+See [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.

+

+## Exported data / Validate data export {#exported-data}

+

+For [!DNL Pega Profile Connector] destinations, [!DNL Platform] creates a `.csv` file in Amazon S3 storage location that you provided. For more information about the files, see [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) in the segment activation tutorial.

+

+A successful import of profile data from S3 inserts data in the Pega Customer profile datastore. The imported Customer profile data can be validated in Pega Customer Profile Designer , as shown in the following figure.

+![Image of the UI screen where you can validate Adobe profile data in Customer Profile Designer](../../assets/catalog/personalization/pega-profile/pega-profile-data.png)

+

+In Pega Customer Decision Hub, Data Administrators can configure data jobs in Customer Profile Designer to import profile data periodically from S3 as shown in the following figure.

also, I believe Data Administrators can be lowercase. unless it is a branded term in Pega?

________________________________

In help/destinations/catalog/personalization/pega-profile.md<https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1014471299>:

> +>In the connect destination workflow, you can create a custom folder in your Amazon S3 storage per exported segment file. Read [Use macros to create a folder in your storage location](overview.md#use-macros) for instructions.

+

+### Enable alerts {#enable-alerts}

+

+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).

+

+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.

+

+## Activate segments to this destination {#activate}

+

+>[!IMPORTANT]

+>

+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.

+

+See [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.

+

there is a subsection which would come in handy here, about the mapping step. see the self-service documentation template:

https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#map<https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#map>

Could we perhaps add this section and briefly describe the particularities of the mapping step for this new Pega destination?

—
Reply to this email directly, view it on GitHub<https://github.com/AdobeDocs/experience-platform.en/pull/131#pullrequestreview-1169188164>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AU3DX2DZGLCD7TIMVMR33FTWGV4E7ANCNFSM6AAAAAARPSZTFI>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,136 @@
+
+## Additional resources {#additional-resources}
+
+See [Import data jobs](https://academy.pega.com/topic/import-data-jobs/v1)) in [!DNL Pega Customer Decision Hub].
```
Really minor, there is a trailing `)` in the link.

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,136 @@
+
+Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the [!DNL Pega Cus ...
+
+This connector helps setup initial export of profile data and also helps sync up new profiles periodically into [!DNL Pega Customer Decision Hub].  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.
```
minor, but setup should actually be two words - set up

paratAtPega replied:
Done

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,136 @@
+
+Use the [!DNL Pega Profile Connector] in Adobe Experience Platform to create a live outbound connection to your [!DNL Amazon Web Services] (AWS) S3 storage to periodically export profile data to CSV files from Adobe Experience Platform into your own S3 buckets. In [!DNL Pega Customer Decision Hub], you can schedule data jobs to import this profile data from S3 storage to update the [!DNL Pega Cus ...
+
+This connector helps setup initial export of profile data and also helps sync up new profiles periodically into [!DNL Pega Customer Decision Hub].  Having up-to-date data in Customer Decision Hub provides a better and updated view of your customer base for next-best-action decisioning.
```
minor, but setup should actually be two words - set up. also, a "the" is missing, as in "set up the initial export [...]"

vgiurgiu commented on help/destinations/catalog/personalization/pega-profile.md:
```diff
@@ -0,0 +1,136 @@
+
+### Fill in destination details {#destination-details}
+
+After establishing the authentication connection to the [!DNL Amazon S3], provide the following information for the destination:
```
minor: After establishing the authentication connection to ~the~ [!DNL Amazon S3]

vgiurgiu reviewed (commented):
Hi @paratAtPega, apologies for the late reply. Thanks for the updates! I left some more minor comments, in addition to a more extensive one about adding information for the mapping step (see comment [here](https://github.com/AdobeDocs/experience-platform.en/pull/131#discussion_r1022089767))

paratAtPega committed:
Update pega-profile.md

paratAtPega committed:
Update pega-profile.md
